### PR TITLE
feat(general): allow listing and confirming users manually. Fixes MA-77

### PR DESCRIPTION
**Pull request by serge1peshcoff**
a) added 2 endpoints: 
1) /members/unconfirmed - list unconfirmed users
2) /members/:id/confirm - confirm users
b) added permissions for seeding
c) added tests

### CODE REVIEW

File: test/api/users-confirmation.test.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        expect(res.body.success).toEqual(false);
+        expect(res.body).not.toHaveProperty('data');
+        expect(res.body).toHaveProperty('message');
+    });
+
+    test('should succeed if everything is okay', async () => {
+        const user = await generator.createUser({ superadmin: true });
+        const token = await generator.createAccessToken({}, user);
+
+        await generator.createPermission({ scope: 'local', action: 'confirm', object: 'member' });
```
Comment (WikiRik) [inline test/api/users-confirmation.test.js:40]: Why is the scope of the permission local?

**Review by WikiRik — APPROVED** [review]
Left one question, but lgtm

File: test/api/users-confirmation.test.js (hunk 1 above)
Comment (serge1peshcoff) [reply test/api/users-confirmation.test.js]: ah yeah, accessing it with local permission doesn't make sense, as the listing is only accessed with the local one

**Pull request merged**